src/components/about/HISS.jsx

```diff
@@ -1,7 +1,49 @@
 import React from "react";
+import Header from "../Header.jsx"; // previously was: src/components/Header
+import Image from "next/image";
+import About from "../../../public/images/about.png";
 
 const HISS = () => {
-  return <div>HISS</div>;
+  return (
+    <div className="flex-col items-center justify-center ml-1 inline-flex">
+      <Header text="About HISS" />
+      <p>
+        The Highlander Statistics Society (HiSS) is a student organization whose
+        <br></br>
+        purpose is to form unity and friendship among students majoring or
+        <br></br>
+        interested in statistics. HiSS works to promote scholarship and interest
+        in<br></br>
+        statistics while informing people about opportunities and challenges
+        that are<br></br>
+        present in the field. Through social, academic, and service events, HiSS
+        strives<br></br>
+        to build a strong community among its club members in addition to
+        <br></br>
+        encouraging involvement and success in statistics.
+      </p>
+      <Image
+        className="rounded-full w-4/6 h-1/3 py-5 px-10"
```

Comment:
1. px-10 is a little bit too much
2. rounded-full is a little too much, try looking at different options https://tailwindcss.com/docs/border-radius